### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by JATINLALITMANDAL** [zhongyang219/TrafficMonitor#954] [General]
After rebooting the system, it is working as the old version instead of the updated version. And I am unable to understand, how to activate the updated version because after downloading the file when I am trying to install it by double-clicking the .exe file it is showing "This is already an instance is running". Though there is no option to uninstall the old version so, I am unable to uninstall the older one and install/run the newer one.

Please give me a solution. Thanks

**Comment by Sunovn** [comment]
same problem. I want to update it **with** my settings and history.

**Reply by smkrsajid** [reply]
I'm stuck too. want to uninstall it cause it's in the middle of my app icons, not where it should be. Any fix devs?

**Comment by Dickmaulruessler** [comment]
I don ´t understand the update process. 
Ich have downloaded the file. But what do I do now? 
I want to keep my old settings and traffic history.

**Reply by airpull91** [reply]
same here..im trying manually transfer the zip file find the folder on my C:\Program Files but couldn't find any

**Comment by sewooru** [comment]
설치프로그램이 아니니...
1. 작동중인 TrafficMonitor 프로그램을 종료합니다.
2. 기존 설치된 TrafficMonitor폴더에 config.ini 바탕화면에 복사>백업해 둡니다.. (사용자설정파일)
3. 기존 TrafficMonitor 폴더 내용을 삭제 합니다. 
4. 다운받은 파일 압축을 풀어줍니다. 
5. 복사해둔 config.ini  여기다 넣어 줍니다.
6. *.EXE 실행한다.
8. 바뀐게 없어보여도 옵션가면 바뀐게 있긴합니다.
<img width="221" height="67" alt="스크린샷 2026-03-31 110230" src="https://github.com/user-attachments/assets/d1580d65-c6ff-4ff2-8f07-57368d59a3c3" />

신규 기능에  테스크바 배경색을 투명해주는게 추가되어 적용시켜 봤습니다. 
여전히 메인보드 온도는 표시되지 않고있네요.
